### PR TITLE
Latest pl_PL for 3.x - ojs

### PR DESCRIPTION
**Pull request by szmigieldesign**
Source: http://szmigiel.design/files/ojs-31-pl_PL-19-03-2018.tar.gz

### CODE REVIEW

File: plugins/generic/htmlArticleGalley/locale/pl_PL/locale.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
```
Comment (mtub) [inline plugins/generic/htmlArticleGalley/locale/pl_PL/locale.xml]: should be ../../../../../

File: plugins/generic/openAIRE/locale/pl_PL/locale.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
```
Comment (mtub) [inline plugins/generic/openAIRE/locale/pl_PL/locale.xml]: see above, ../../../../../

File: plugins/generic/openAIRE/locale/pl_PL/locale.xml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+<!--
+  * locale/pl_PL/locale.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University
```
Comment (mtub) [inline plugins/generic/openAIRE/locale/pl_PL/locale.xml]: Copyright

File: plugins/generic/recommendByAuthor/locale/pl_PL/locale.xml [hunk 4]
```diff
@@ -4,8 +4,8 @@
 <!--
   * plugins/generic/recommendByAuthor/locale/pl_PL/locale.xml
   *
-  * Copyright (c) 2014-2018 Simon Fraser University
-  * Copyright (c) 2003-2018 John Willinsky
+  * Copyright (c) 2014-2017 Simon Fraser University
```
Comment (mtub) [inline plugins/generic/recommendByAuthor/locale/pl_PL/locale.xml]: Copyright

File: plugins/generic/usageEvent/locale/pl_PL/locale.xml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
```
Comment (mtub) [inline plugins/generic/usageEvent/locale/pl_PL/locale.xml]: see above, ../../../../../

File: plugins/generic/usageEvent/locale/pl_PL/locale.xml [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+<!--
+  * locale/pl_PL/locale.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University
```
Comment (mtub) [inline plugins/generic/usageEvent/locale/pl_PL/locale.xml]: Copyright

File: plugins/generic/webFeed/locale/pl_PL/locale.xml [hunk 7]
```diff
@@ -4,8 +4,8 @@
 <!--
   * plugins/generic/webFeed/locale/pl_PL/locale.xml
   *
-  * Copyright (c) 2014-2018 Simon Fraser University
-  * Copyright (c) 2003-2018 John Willinsky
+  * Copyright (c) 2014-2017 Simon Fraser University
```
Comment (mtub) [inline plugins/generic/webFeed/locale/pl_PL/locale.xml]: Copyright

File: plugins/importexport/crossref/locale/pl_PL/locale.xml [hunk 8]
```diff
@@ -4,8 +4,8 @@
 <!--
   * plugins/importexport/crossref/locale/pl_PL/locale.xml
   *
-  * Copyright (c) 2014-2018 Simon Fraser University
-  * Copyright (c) 2003-2018 John Willinsky
+  * Copyright (c) 2014-2017 Simon Fraser University
```
Comment (mtub) [inline plugins/importexport/crossref/locale/pl_PL/locale.xml]: Copyright

File: plugins/importexport/crossref/locale/pl_PL/locale.xml [hunk 9]
```diff
@@ -16,27 +16,47 @@
...
+	<message key="plugins.importexport.crossref.action.export">Pobierz XML</message>
+	<message key="plugins.importexport.crossref.action.markRegistered">Oznacz jako aktywne</message>
+	<message key="plugins.importexport.crossref.action.register">Prześlij</message>
+	<message key="plugins.importexport.crossref.action.checkStatus">Sprawdź status</message>
+	<message key="plugins.importexport.crossref.notification.failed"><![CDATA[Błąd rejestracji DOI. Przejdź do Narzędzia -> Import/Eksport > CrossRef XML aby sprawdzić błędy.]]></message>
+	<message key="plugins.importexport.crossref.register.error.mdsError">Przesyłanie nieudane! Serwer DOI zwrócił błąd: '{$param}'.</message>
+	<message key="plugins.importexport.crossref.register.success">Przesyłanie zakończone sukcesem!</message>
+	<message key="plugins.importexport.crossref.export.error.issueNotFound">Żaden numer nie pasuje do ID "{$issueId}".</message>
+	<message key="plugins.importexport.crossref.export.error.articleNotFound">Żaden tekst nie pasuje do ID "{$articleId}".</message>
+	<message key="plugins.importexport.crossref.statusLegend"><![CDATA[<p>Status:</p>
```
Comment (mtub) [inline plugins/importexport/crossref/locale/pl_PL/locale.xml]: English text

File: plugins/importexport/datacite/locale/pl_PL/locale.xml [hunk 10]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
```
Comment (mtub) [inline plugins/importexport/datacite/locale/pl_PL/locale.xml]: see above, ../../../../../

File: plugins/importexport/datacite/locale/pl_PL/locale.xml [hunk 11]
```diff
@@ -0,0 +1,27 @@
...
+
+<!--
+  * locale/pl_PL/locale.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University
+  * Copyright (c) 2003-2017 John Willinsky
```
Comment (mtub) [inline plugins/importexport/datacite/locale/pl_PL/locale.xml]: Copyright

File: plugins/importexport/doaj/locale/pl_PL/locale.xml [hunk 12]
```diff
@@ -4,17 +4,28 @@
 <!--
   * plugins/importexport/doaj/locale/pl_PL/locale.xml
   *
-  * Copyright (c) 2013-2018 Simon Fraser University
-  * Copyright (c) 2003-2018 John Willinsky
+  * Copyright (c) 2013-2017 Simon Fraser University
```
Comment (mtub) [inline plugins/importexport/doaj/locale/pl_PL/locale.xml]: Copyright

File: plugins/importexport/medra/locale/pl_PL/locale.xml [hunk 13]
```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
```
Comment (mtub) [inline plugins/importexport/medra/locale/pl_PL/locale.xml]: see above, ../../../../../

File: plugins/importexport/medra/locale/pl_PL/locale.xml [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+
+<!--
+  * locale/pl_PL/locale.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University
```
Comment (mtub) [inline plugins/importexport/medra/locale/pl_PL/locale.xml]: Copyright

File: plugins/importexport/native/locale/pl_PL/locale.xml [hunk 15]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale/pl_PL/locale.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University
```
Comment (mtub) [inline plugins/importexport/native/locale/pl_PL/locale.xml]: Copyright (also all other files, will stop here… ;))

File: plugins/importexport/native/locale/pl_PL/locale.xml [hunk 16]
```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
```
Comment (mtub) [inline plugins/importexport/native/locale/pl_PL/locale.xml]: see above, ../../../../../

File: plugins/importexport/pubmed/locale/pl_PL/locale.xml [hunk 17]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
```
Comment (mtub) [inline plugins/importexport/pubmed/locale/pl_PL/locale.xml]: see above, ../../../../../

File: plugins/metadata/mods34/locale/pl_PL/locale.xml [hunk 18]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
```
Comment (mtub) [inline plugins/metadata/mods34/locale/pl_PL/locale.xml]: see above, ../../../../../

File: plugins/oaiMetadataFormats/rfc1807/locale/pl_PL/locale.xml [hunk 19]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
```
Comment (mtub) [inline plugins/oaiMetadataFormats/rfc1807/locale/pl_PL/locale.xml]: see above, ../../../../../

File: plugins/oaiMetadataFormats/marcxml/locale/pl_PL/locale.xml [hunk 20]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
```
Comment (mtub) [inline plugins/oaiMetadataFormats/marcxml/locale/pl_PL/locale.xml]: see above, ../../../../../

File: plugins/paymethod/manual/locale/pl_PL/emailTemplates.xml [hunk 21]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE email_texts SYSTEM "../../../../lib/pkp/dtd/emailTemplateData.dtd">
```
Comment (mtub) [inline plugins/paymethod/manual/locale/pl_PL/emailTemplates.xml]: see above, ../../../../../

File: plugins/paymethod/manual/locale/pl_PL/locale.xml [hunk 22]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
```
Comment (mtub) [inline plugins/paymethod/manual/locale/pl_PL/locale.xml]: see above, ../../../../../

File: plugins/paymethod/paypal/locale/pl_PL/emailTemplates.xml [hunk 23]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE email_texts SYSTEM "../../../../lib/pkp/dtd/emailTemplateData.dtd">
```
Comment (mtub) [inline plugins/paymethod/paypal/locale/pl_PL/emailTemplates.xml]: see above, ../../../../../

File: plugins/paymethod/paypal/locale/pl_PL/locale.xml [hunk 24]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
```
Comment (mtub) [inline plugins/paymethod/paypal/locale/pl_PL/locale.xml]: see above, ../../../../../

File: plugins/pubIds/urn/locale/pl_PL/locale.xml [hunk 25]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
```
Comment (mtub) [inline plugins/pubIds/urn/locale/pl_PL/locale.xml]: see above, ../../../../../

File: plugins/reports/counter/locale/pl_PL/locale.xml [hunk 26]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
```
Comment (mtub) [inline plugins/reports/counter/locale/pl_PL/locale.xml]: see above, ../../../../../

File: plugins/reports/views/locale/pl_PL/locale.xml [hunk 27]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../lib/pkp/dtd/locale.dtd">
```
Comment (mtub) [inline plugins/reports/views/locale/pl_PL/locale.xml]: see above, ../../../../../

**Comment by szmigieldesign** [comment]
I've fixed dtds and dates. Unfortunately I don't have time at present to work on missing translations, especially considering they sit in CDATA which requires thorough translation process. Should I request new pull?

**Comment by mtub** [comment]
Thank you, @szmigieldesign - there are some remaining issues (wrong DTD links) that I will fix later in a separate commit. Your contribution is very much appreciated.

**Comment by szmigieldesign** [comment]
Sorry for wrong DTDs. I believe that we worked on v3.0 version and they must have stuck. Now that I use git, future updates should be less of a problem.

**Pull request merged**